### ISSUE TITLE
Accessibility | Focus problem with the navigation

### ISSUE THREAD

**Issue by f4-lex** [Frachtwerk/essencium-frontend#831]
**Describe the bug**
When the navigation bar isn't pinned, it is impossible to reach any subitems with keyboard navigation only. 

**Expected behaviour**
All navigation items should be reachable by keyboard navigation.

- [ ] update `MIGRATION.md`
- [ ] update docs
